docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
```

Comment:
Maybe change to
```suggestion
/build/
```
to avoid that other directories named build are excluded. Possibly also just add it to the main .gitignore?